index.js

```diff
@@ -111,9 +122,13 @@ module.exports = (bookshelf, settings) => {
           // Hash the password when saving
           this.on('saving', (model, attrs, options) => {
             let field = get(this, 'bcrypt.field')
-
             if (model.hasChanged(field) && options.bcrypt !== false) {
-              return hashAndStore(model.get(field), field, model)
+              const password = model.get(field)
```

Comment:
Will do. Just curious, why do you opt for `let` over `const` for something that's not reassigned?